src/daemons/linuxd/src/venv.rs

```diff
@@ -23,6 +23,20 @@ use ::sys::{
 };
 use ::syscall::venv::VirtualEnvironmentIdentifier;
 
+///
+/// # Description
+///
+/// Unique identifier for each user VM.
+///
+pub type UserVmIdentifier = usize;
```

Comment:
Consider using `u32`  as `usize` is not portable.

Reply:
I chose `usize` because that is the key value for `Slab`, so I was using it to index the different data structures. In the long run it is better to have `u32`, because we, at some point, will start indexing things with the user VM ID assigned by nanvixd, not the connection id.

In the last commit I change the indexing to use u32, and convert when appropriate.